marketplace/netlify-build/extension.json

```diff
@@ -0,0 +1,53 @@
+{
+  "id": "netlify-button",
+  "name": "Netlify Build and Preview",
+  "src": "http://localhost:1234",
+  "sidebar": true,
+  "parameters": {
+    "installation": [
+      {
+        "id": "publishKey",
+        "type": "Symbol",
+        "name": "PubNub publish key",
+        "description": "A public PubNub publish key to publish update events to Netlify",
+        "required": true
+      },
+      {
+        "id": "subscribeKey",
+        "type": "Symbol",
+        "name": "PubNub subscribe key",
+        "description": "A public PubNub subscribe key to listen for update events from Netlify",
+        "required": true
+      },
+      {
+        "id": "channels",
+        "type": "Symbol",
+        "name": "Comma seperated Netlify channel list (Max 3)",
```

Comment:
These are PubNub channels. Would you capitalize "max"?

Reply:
I would but since we already show `max 3` in the main Netlify configuration page, I will change it to lowercase to keep consistency.